src/Toast/toast.tsx

```diff
@@ -29,4 +32,5 @@ const toast = INTENTS.reduce(
   {} as Record<Intent, ToastFunction>
 );
 
+export type ToastOptions = ReactToastifyToastOptions;
```

Comment:
I would move this above `type ToastFunction`, and then `toastifyOptions?` can stay as `ToastOptions`.

Reply:
Oh niiice.